.github/workflows/build_and_test.yml

```diff
@@ -1,13 +1,13 @@
 name: build-and-test
 
-on: [push, pull_request]
+on: [push]
 
 env:
   CARGO_TERM_COLOR: always
   THOTH_API: https://api.thoth.openbookpublishers.com
 
 jobs:
-  build_and_test:
+  build:
```

Comment:
Rename back to `build_and_test` (ditto below - or perhaps that one should be `build_test_and_lint`)